sdk/python/pulumi_kubernetes/yaml.py

```diff
@@ -0,0 +1,634 @@
+# *** WARNING: this file was generated by the Pulumi Kubernetes codegen tool. ***
+# *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+import json
+from copy import deepcopy
+from typing import Optional
+
+import pulumi
+import pulumi.runtime
+import requests
+import yaml
+from pulumi_kubernetes.admissionregistration.v1alpha1 import *
+from pulumi_kubernetes.admissionregistration.v1beta1 import *
+from pulumi_kubernetes.apiextensions.v1beta1 import *
+from pulumi_kubernetes.apiregistration.v1 import *
+from pulumi_kubernetes.apiregistration.v1beta1 import *
+from pulumi_kubernetes.apps.v1 import *
+from pulumi_kubernetes.apps.v1beta1 import *
+from pulumi_kubernetes.apps.v1beta2 import *
+from pulumi_kubernetes.auditregistration.v1alpha1 import *
+from pulumi_kubernetes.authentication.v1 import *
+from pulumi_kubernetes.authentication.v1beta1 import *
+from pulumi_kubernetes.authorization.v1 import *
+from pulumi_kubernetes.authorization.v1beta1 import *
+from pulumi_kubernetes.autoscaling.v1 import *
+from pulumi_kubernetes.autoscaling.v2beta1 import *
+from pulumi_kubernetes.autoscaling.v2beta2 import *
+from pulumi_kubernetes.batch.v1 import *
+from pulumi_kubernetes.batch.v1beta1 import *
+from pulumi_kubernetes.batch.v2alpha1 import *
+from pulumi_kubernetes.certificates.v1beta1 import *
+from pulumi_kubernetes.coordination.v1beta1 import *
+from pulumi_kubernetes.core.v1 import *
+from pulumi_kubernetes.events.v1beta1 import *
+from pulumi_kubernetes.extensions.v1beta1 import *
+from pulumi_kubernetes.meta.v1 import *
+from pulumi_kubernetes.networking.v1 import *
+from pulumi_kubernetes.policy.v1beta1 import *
+from pulumi_kubernetes.rbac.v1 import *
+from pulumi_kubernetes.rbac.v1alpha1 import *
+from pulumi_kubernetes.rbac.v1beta1 import *
+from pulumi_kubernetes.scheduling.v1alpha1 import *
+from pulumi_kubernetes.scheduling.v1beta1 import *
+from pulumi_kubernetes.settings.v1alpha1 import *
+from pulumi_kubernetes.storage.v1 import *
+from pulumi_kubernetes.storage.v1alpha1 import *
+from pulumi_kubernetes.storage.v1beta1 import *
+
+from . import tables
+
+
+class ConfigFile(pulumi.ComponentResource):
+    """
+    ConfigFile creates a set of Kubernetes resources from a Kubernetes YAML file. If `config.name`
+    is not specified, `ConfigFile` assumes the argument `name` is the filename.
+    """
+
+    def __init__(self, __name__: str, file_id: str, __opts__: Optional[pulumi.ResourceOptions] = None):
```

Comment:
It's unfortunate that we double-undescore `__name__` and `__opts__` here - it throws Sphinx for a loop, Can you drop the double underscores? (We should also drop the double underscores for everything else at some point too.)